### DISCUSSION TITLE
Node.JS 16, is there a removal date?

### DISCUSSION THREAD

**Discussion by odelaere** [actions/virtual-environments#9390] [Installed software]
Hello,

Node.JS 16 reached its end of life on September 2023.
Is there any planed date to remove it ?

Regards

**Comment by v-sergei-pyshnoi** [comment]
Hello @odelaere . Due runner-images [support policy for NodeJS](https://github.com/actions/runner-images?tab=readme-ov-file#software-and-image-support) we keep 3 latest LTS version. So NodeJS will be removed with announce after next LTS will come out.
